source/scripting/VehicleDoor.cs

```diff
@@ -75,7 +75,11 @@ public void Close(bool instantly)
 		}
 		public void BreakOff()
 		{
-			Function.Call(Hash.SET_VEHICLE_DOOR_BROKEN, _owner.Handle, (int)Index);
+			Function.Call(Hash.SET_VEHICLE_DOOR_BROKEN, _owner.Handle, (int)Index, true);
+		}
+		public void Delete()
+		{
+			Function.Call(Hash.SET_VEHICLE_DOOR_BROKEN, _owner.Handle, (int)Index, false);
```

Comment:
If the 3rd param is whether the door stays in the world, the true and false should be switched